### DISCUSSION TITLE
Where can I find a good qwerty config for meow?

### DISCUSSION THREAD

**Discussion by jgarte** [meow-edit/meow#332] [Q&A]
Hi, I don't have time at the moment to completely build out a whole meow config for qwerty. Is there anyone that has a package that configures meow to work similar to what I expect from vim?

**Comment by eshrh** [comment]
The de-facto one is here: https://github.com/meow-edit/meow/blob/master/KEYBINDING_QWERTY.org. 

It is still very different to vim in a lot of ways though, but that's kind of unavoidable. The linked keybinds lean heavily on mnemonic letters, so they shouldn't be terribly difficult to pick up. I'd also read https://github.com/meow-edit/meow/blob/master/VIM_COMPARISON.org.

I'm sure someone's hacked a faithful set of vim keybinds by now though.
